packages/@webex/plugin-meetings/test/unit/spec/multistream/remoteMediaManager.ts

```diff
@@ -663,6 +663,36 @@ describe('RemoteMediaManager', () => {
       remoteMediaManager.stop();
     });
   });
+
+  describe('setPreferLiveVideo', () => {
+
+    it('sets preferLiveVideo', async () => {
+      const config = cloneDeep(DefaultTestConfiguration);
+
+      config.video.initialLayoutId = 'OnePlusFive';
+
+      remoteMediaManager = new RemoteMediaManager(
+        fakeReceiveSlotManager,
+        fakeMediaRequestManagers,
+        config
+      );
+  
+      await remoteMediaManager.start();
+  
+      resetHistory();
+      remoteMediaManager.setPreferLiveVideo(true);
+
+      expect(config.video.preferLiveVideo).to.equal(true);
+      
+      remoteMediaManager.on(Event.VideoLayoutChanged, (layoutInfo: VideoLayoutChangedEventData) => {
+        Object.values(layoutInfo.activeSpeakerVideoPanes).forEach((group) =>
+          assert.calledOnceWithExactly(group.setPreferLiveVideo, true, false)
```

Comment:
I don't think this assert is ever executed. You need to register the listener for `Event.VideoLayoutChanged` before calling `start()`, because that's when that event is emitted.